### PR TITLE
add Armistead_CellDeathDis2024

### PR DESCRIPTION
**Pull request by m-philipps**
**Checklist for the submission of new PEtab problems**
- [x] The PEtab problem is based on a model that is peer-reviewed and published
- [x] The problem ID is in the format `{LAST_NAME_OF_FIRST_AUTHOR}_{ABBREVIATED_JOURNAL_NAME}{YEAR_OF_PUBLICATION}`
- [x] The problem ID is in the pull request title
- [x] There is a GitHub issue for this problem (#247), #248
  - [x] The problem ID is in the issue title
  - [x] A brief model description (one or two sentences)
  - [x] A brief data description (one or two sentences)
  - [x] The issue and PR are linked to each other
  - [x] Differences between the implementation and the original publication are described
  - [x] Experience of fitting / uncertainty analysis (e.g. optimizer used, hyperparameters, reproducibility of best fit)
  - [x] Source of nominal parameters (e.g.: taken from the original publication, or from your own fitting)
- [x] The SBML file
  - [x] Annotation with reference to the original publication ([example](https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/blob/3154d27edbfc4eb19b768465945bc81381a7c964/Benchmark-Models/Rahman_MBS2016/model_Rahman_MBS2016.xml#L4-L14))
  - [x] The model ID and model name attributes in the SBML model file match the problem name ([example](https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/blob/3154d27edbfc4eb19b768465945bc81381a7c964/Benchmark-Models/Rahman_MBS2016/model_Rahman_MBS2016.xml#L3))
- [x] PEtab files
  - [x] A "simulated data" measurement table is included, using the nominal parameters
  - [X] A visualization table is included, that can be used with the simulated data to reproduce figures from the original publication
  - [x] The PEtab problem is valid (check with e.g. `petablint -vy problem.yaml`)
- [x] The PEtab problem author(s) are assigned to the GitHub issue
- [x] The README has been updated with `bmp-create-overview  --update` (requires `pip install -e src/python/bmp` from the repository root)
  - [x] The new PEtab problem row in the generated table has the correct reference (and other entries)

### CODE REVIEW

File: Benchmark-Models/Armistead_CellDeathDis2024/simulationData_Armistead_CellDeathDis2024.tsv
Comment (dilpath) [inline Benchmark-Models/Armistead_CellDeathDis2024/simulationData_Armistead_CellDeathDis2024.tsv:1]: Ideally without the index (like a PEtab measurements table), but fine as-is

File: Benchmark-Models/Armistead_CellDeathDis2024/visualizationSpecification_Armistead_CellDeathDis2024.tsv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+plotId	plotName	plotTypeData	xLabel	yValues	yLabel	yScale
+plot1	highandlow	MeanAndSD	Time	Sphinga_obs	Concentration [area ratio/Protein]	log10
```
Comment (dilpath) [inline Benchmark-Models/Armistead_CellDeathDis2024/visualizationSpecification_Armistead_CellDeathDis2024.tsv:2]: Change the `plotId` to reference the reproduced figure from the paper, e.g. `plotId=Fig1A`. If a figure from the paper is not reproducible via the visualization table, then uncheck this point in the checklist and add a note to the issue.

**Review by dilpath — APPROVED** [review]
Thanks very much! Everything looks mostly great, including the details in the issue. Feel free to merge.

File: Benchmark-Models/Armistead_CellDeathDis2024/visualizationSpecification_Armistead_CellDeathDis2024.tsv (hunk 1 above)
Comment (m-philipps) [reply Benchmark-Models/Armistead_CellDeathDis2024/visualizationSpecification_Armistead_CellDeathDis2024.tsv:2]: The information from Figure 4 k) can be reproduced, but the visualisation specification is not flexible enough to reproduce the Figure visually.

Comment (dilpath) [reply Benchmark-Models/Armistead_CellDeathDis2024/visualizationSpecification_Armistead_CellDeathDis2024.tsv]: I made a new vis spec file to show Fig4k, please review and then merge if approved

Comment (dilpath) [reply Benchmark-Models/Armistead_CellDeathDis2024/visualizationSpecification_Armistead_CellDeathDis2024.tsv]: image

**Pull request merged**